scripts/ci/metrics.sh

```diff
@@ -209,3 +209,62 @@ LIMIT
     echo "Posting data to slack"
     jq -n --arg data "$data" "$body" | curl -XPOST -d @- -H 'Content-Type: application/json' "$webhook_url"
 }
+
+# Saving test metrics directly after tests complete results in GCP quota issues.
+# So instead they are saved to GCS and dealt with in batches as per the remedy
+# in https://cloud.google.com/bigquery/docs/troubleshoot-quotas#ts-table-import-quota-resolution
+
+_TESTS_TABLE_NAME="acs-san-stackroxci:ci_metrics.stackrox_tests"
+_BATCH_STORAGE_UPLOAD="gs://stackrox-ci-artifacts/test-metrics/upload"
+_BATCH_STORAGE_PROCESSING="gs://stackrox-ci-artifacts/test-metrics/processing"
+_BATCH_SIZE=20
```

Comment:
Just curious how did you arrive at this number? Why not something larger? 100?

Reply:
> Just curious how did you arrive at this number? Why not something larger? 100?

I'd like to keep it small so if there are load failures the dataset will be easier to manage. But large enough to work within GCP quota.